packages/styles/scss/components/_video.scss

```diff
@@ -4,6 +4,8 @@
 
 .ilo--video {
   position: relative;
+  width: 100%;
+  max-width: 855px;
```

Comment:
@PauZoocha why is this needed? if you want the player to have max-width, just wrap it in a `div` no?